main.tf

```diff
@@ -34,7 +34,7 @@ data "aws_iam_policy_document" "assume_role" {
 
 resource "aws_iam_role" "default" {
   count              = var.enabled ? 1 : 0
-  name               = module.label.id
+  name               = "${module.label.id}-${var.kubernetes_namespace}"
```

Comment:
@Nuru is this where we should use `${module.label.id}@${var.kubernetes_namespace}` following our current convention?